api/features/views.py

```diff
@@ -636,6 +637,31 @@ def all(self, request, *args, **kwargs):
 
         return Response(serializer.data)
 
+    @swagger_auto_schema(request_body=IdentitySourceIdentityRequestSerializer())
+    @action(methods=["POST"], detail=False, url_path="clone-from-given-identity")
+    def clone_from_given_identity(self, request, *args, **kwargs) -> Response:
+        """
+        Clone feature states from a given source identity.
+        """
+        serializer = IdentitySourceIdentityRequestSerializer(
+            data=request.data, context={"request": request}
+        )
+        serializer.is_valid(raise_exception=True)
+        # Get and validate source and target identities
+        target_identity = get_object_or_404(
+            queryset=Identity, pk=self.kwargs["identity_pk"]
+        )
+        source_identity = get_object_or_404(
+            queryset=Identity, pk=request.data.get("source_identity_id")
+        )
+
+        # Clone feature states
+        FeatureState.copy_identity_feature_states(
+            target_identity=target_identity, source_identity=source_identity
+        )
+
+        return Response(status=status.HTTP_200_OK)
```

Comment:
Again, we should return some data here - the target identity and their overrides perhaps? I guess we want to return whatever data the FE needs to re-render the identity detail page.

Reply:
This is answered in the same observation for the Core endpoint.

Reply:
Right. Data being returned again.